src/main/java/com/alkemy/ong/dto/UserRegisterRequest.java

```diff
@@ -0,0 +1,39 @@
+package com.alkemy.ong.dto;
+
+import com.alkemy.ong.model.Role;
+import javax.validation.constraints.*;
+
+import com.alkemy.ong.model.User;
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.util.List;
+import lombok.*;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserRegisterRequest {
+
+    @NotBlank(message = "Name is mandatory.")
+    private String firstName;
+
+    @NotBlank(message = "Last name is mandatory.")
+    private String lastName;
+
+    @NotBlank(message = "Email is mandatory.")
+    @Email(message = "Email should be valid.", regexp = "[A-Za-z0-9._%-+]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,4}")
+    private String email;
+
+    @NotBlank(message = "Password is mandatory.")
+    @Size(min = 6, message = "Password must be between 6")
```

Comment:
Una pavada pero el menasje de error está mal, between significa entre, y la validación dice que es un mínimo de 6 caracteres, por lo que debería decir algo como: "The password must be at least 6 characters"